web-console/4.0-designs/overview/overview.md

```diff
@@ -0,0 +1,87 @@
+# Overview
+
+## Collapsed list view
+
+### General
+
+![list view](img/listview.png)
+* The project overview may be viewed in list view or topology view. It will default to show the list view. Users can toggle between the two views by using the view selector icons at the top right corner of the page.
+* Users may filter resources by name by using the filter bar in the top right.
+* The collapsed list view of the project overview will group items by app label (for example, `app=foo`) by default. Users may group by a different label by using the `Group by label` dropdown selector. The selector will be populated with existing labels.
+  * If no resources have labels, items will be shown in a regular list view listed alphabetically by resource type and then name and the `Group by label` dropdown will be hidden.
+* On hover, the row will be highlighted (#EDF8FF) and the icon fa-angle-right (pf-blue-400, #0088CE) will appear at the end of the row. When a row is selected (#DEF3FF), the side panel will open with a summary of that resource's overview, pushing the content over to the left.
+* If a user is not in the context of a selected project, the user will be shown an [empty state](http://openshift.github.io/openshift-origin-design/web-console/4.0-designs/empty-states/empty-states) until they select a project from the project dropdown.
+![empty state](img/ES_overview.png)
```

Comment:
I'd still argue to change the wording of this message (per my comments in the previous doc)